File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/content/GetPagedDocumentsFromFolderRequestModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.content;

public class GetPagedDocumentsFromFolderRequestModel {

	private String documentLocationRealName;
	private String folderId;
	private boolean sameType;
	
	public String getDocumentLocationRealName() {
		return documentLocationRealName;
	}
	public void setDocumentLocationRealName(String documentLocationRealName) {
		this.documentLocationRealName = documentLocationRealName;
	}
	public String getFolderId() {
		return folderId;
	}
	public void setFolderId(String folderId) {
		this.folderId = folderId;
	}
	public boolean isSameType() {
		return sameType;
	}
	public void setSameType(boolean sameType) {
		this.sameType = sameType;
	}
}

File: CloudFlow/FrontEnd-App/src/app/shared/model/get-paged-documents-from-folder-request.model.ts
import { JsonObject, JsonProperty } from "json2typescript";

@JsonObject
export class GetPagedDocumentsFromFolderRequestModel {

	@JsonProperty("documentLocationRealName", String)
	public documentLocationRealName: string = null;

	@JsonProperty("folderId", String)
	public folderId: string = null;

	@JsonProperty("sameType", Boolean)
	public sameType: boolean = null;
}